packages/dds/tree/src/test/simple-tree/treeApi.spec.ts

```diff
@@ -108,6 +108,33 @@ describe("treeApi", () => {
 		// TODO: test Deleted status.
 	});
 
+	describe("shortID", () => {
+		it("returns local id when an identifier fieldkind exists.", () => {
+			const schemaWithIdentifier = schema.object("parent", {
+				identifier: schema.identifier(schema.string),
+			});
+			const nodeKeyManager = createMockNodeKeyManager();
+			const id = nodeKeyManager.stabilizeNodeKey(nodeKeyManager.generateLocalNodeKey());
+			const config = new TreeConfiguration(schemaWithIdentifier, () => ({
+				identifier: id,
+			}));
+
+			const root = getView(config, nodeKeyManager).root;
+
+			assert.equal(Tree.shortID(root), nodeKeyManager.localizeNodeKey(id));
+		});
+		it("returns undefined when an identifier fieldkind does not exist.", () => {
+			const schemaWithIdentifier = schema.object("parent", {
+				identifier: schema.optional(schema.string),
```

Comment:
nit: use required since that more closely matches what `schema.identifier` does.
```suggestion
				identifier: schema.required(schema.string),
```